common/test/db_cache/bok_choy_schema.sql

```diff
@@ -753,17 +805,6 @@ CREATE TABLE `certificates_generatedcertificate` (
   CONSTRAINT `user_id_refs_id_8e23bfe2` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
-DROP TABLE IF EXISTS `circuit_servercircuit`;
```

Comment:
@e-kolpakov @stvstnfrd The tests will not run without the schema/data update. This is because models are removed, but have data in them within the fixtures. I tried just changing the data and putting the schema back into place, but this did not work on its own.